.changeset/loud-paws-impress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/search-select-input': patch
```

Comment:
I think it's `minor` if we want to follow Semantic Versioning as it's a feature.

Reply:
Yes, makes more sense. Thanks for the heads-up! [60739d7](https://github.com/commercetools/ui-kit/pull/2636/commits/60739d7ca2247f6617766ebfc305c5c2a100bb57)